go-sample/playground/index.html

```diff
@@ -337,6 +341,21 @@
 
                   const node = document.querySelectorAll(`.watcher[data-watcher-item=result][data-watcher="${index}"]`)[0]
```

Comment:
Destructuring makes this a little nicer:

```suggestion
                  const [node] = document.querySelectorAll(`.watcher[data-watcher-item=result][data-watcher="${index}"]`)
```